### ISSUE TITLE
NVDA doesn’t announce selected state after selecting the tree item if it's a jupyter notebook

### ISSUE THREAD

**Issue by MaddyDev** [microsoft/vscode#88800]
- VSCode Version: 1.41.1 (user setup)
- OS Version: Windows_NT x64 10.0.18363

Steps to Reproduce:

1. Open a folder which contains a jupyter book.
2. Navigate to the left section of files on the files explorer and expand the books contents.
3. Start NVDA.
4. Navigate till tree item using arrows and select the item when it's a ipynb file.
5. Observe the announcement by screen reader.

Actual:
NVDA doesn’t announce selected state after selecting the tree item such as “TSG021- Get Kubernetes cluster information” .

Expected:
NVDA should announce selected state after selecting the tree item for example “TSG021 - Get Kubernetes cluster information selected” .

User Impact:
Screen reader user will it difficult to navigate the screens as they won’t understand that particular tree item is selected or not.

Attachment for Reference:
7796_A11y_AzureDataStudio_JuypyterBook_ScreenReader.pptx

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Thgis is actually correct, since moving the cursor in the explorer just moves the focus not the selection.
We have discussed this with our accessibility community and feedback was positive.
This is aslo captured in another issue. Thus closing as wont fix
